src/encord_active/app/model_quality/sub_pages/metrics.py

```diff
@@ -88,12 +88,19 @@ def build(
 
             metric_columns = list(get_state().predictions.metric_datas.predictions.keys())
             with st.spinner("Computing index importance..."):
-                chart = create_metric_importance_charts(
-                    model_predictions,
-                    metric_columns=metric_columns,
-                    num_samples=num_samples,
-                )
-            st.altair_chart(chart, use_container_width=True)
+                try:
+                    chart = create_metric_importance_charts(
+                        model_predictions,
+                        metric_columns=metric_columns,
+                        num_samples=num_samples,
+                    )
+                    st.altair_chart(chart, use_container_width=True)
+                except ValueError as e:
+                    __import__("ipdb").set_trace()
```

Comment:
probably without this right? 🙈

Reply:
Stupid me. I should add an automatic check for that! Fixed